scripts/src/main/resources/scripts/command/docker

```diff
@@ -36,6 +36,10 @@ function doResolveDownloadUrl() {
 
 # $1: optional setup
 function doSetup() {  
+  if doIsWsl1Used; then
+    #update from wsl1 to wsl2, else the following docker editions won't work
+    wsl --update
```

Comment:
Don`t you need admin persmissions to run this?

Reply:
No, at least not if there are no applyable updates available. 
So I can just execute `wsl --update` in a git-bash prompt without admin rights.

Reply:
This is because you already have wsl2 so this command is doing pretty much nothing.
We need to properly test this otherwise we are publishing and untested feature that will most probably never work.